flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/plan/stream/sql/WindowTableFunctionTest.scala

```diff
@@ -86,6 +87,17 @@ class WindowTableFunctionTest extends TableTestBase {
     util.verifyRelPlan(sql)
   }
 
+  @Test
+  def testCumulateTVFProctime(): Unit = {
+    val sql =
+      """
+        |SELECT *
+        |FROM TABLE(
+        | CUMULATE(TABLE MyTable, DESCRIPTOR(proctime), INTERVAL '10' MINUTE, INTERVAL '1' HOUR))
+        |""".stripMargin
+    util.verifyExplain(sql)
```

Comment:
use `util.verifyExecPlan()`

Reply:
I original use util.verifyExecPlan() in order to cover the original `WindowTableFunctionTest#testUnsupported` could work fine now.
However, I think I could use `util.verifyExecPlan()` because `WindowTableFunctionITCase` already cover this.